migration-engine/migration-engine-tests/tests/migration_persistence/imperative_migration_persistence_tests.rs

```diff
@@ -30,11 +30,6 @@ async fn starting_a_migration_works(api: &TestApi) -> TestResult {
     assert_eq!(first_migration.applied_steps_count, 0);
     assert_eq!(first_migration.script, script);
 
-    let duration_since_started_at = chrono::Utc::now().signed_duration_since(first_migration.started_at);
-
-    assert!(duration_since_started_at >= Duration::seconds(0));
-    assert!(duration_since_started_at < Duration::seconds(1));
```

Comment:
I think this assertion has been fixed, is it still failing for you? I'd rather fix it than delete it.

Reply:
The git history of this PR is extremely weird (as pointed out earlier), I assume this is the result of that. Will try to remove that change.